internal/cmd/agent/deployer/summary/summarizers.go

```diff
@@ -353,7 +353,7 @@ func checkInitializing(obj data.Object, conditions []Condition, summary fleetv1.
 	if summary.State == "" && hasConditions && len(conditions) == 0 && strings.Contains(apiVersion, "cattle.io") {
 		val := obj.String("metadata", "created")
 		if i, err := convert.ToTimestamp(val); err == nil {
-			if time.Unix(i/1000, 0).Add(5 * time.Second).After(time.Now()) {
+			if time.Unix(i/1000, 0).Add(5 * time.Second).After(time.Now().UTC()) {
```

Comment:
In Go time includes the time zone.
So, when comparing times,  `if time.Now().UTC().After(expire) {` is no different than `if time.Now().After(expire) {`?

Reply:
Indeed, a few tests reveal that it doesn't make any difference, so we might as well save ourselves the additional method call.